### ISSUE TITLE
Extension Stuck. Only betting on 1 fighter based on old match results.

### ISSUE THREAD

**Issue by shetaron** [FranciscoAT/saltyboy#60]
At current moment in time, the chrome extension is stuck and hyperfixated on an old match and will not bet anything above $1, and always on red. 

![image](https://github.com/FranciscoAT/saltyboy/assets/134396642/328c16ec-e1d9-4ba5-98b2-0aedf7d7500e)

Match results have not updated in 48 hours of the bot running and does not change even on hard reset of machine or uninstall of extension.

**Comment by FranciscoAT** [comment]
Thanks for alerting me! This is due to the upstream servers job that scrapes the twitch chat for fight announcements failing. There's another issue on the project to try and implement some fallback strategies :)

Will kick the bot shortly

**Comment by shetaron** [comment]
I'm gonna be perfectly honest, while I appreciate you replying and telling me what the issue is, I am sitting here looking like:

![image](https://github.com/FranciscoAT/saltyboy/assets/134396642/15013d6e-b2a7-4cf2-aeb8-20cc745880b5)

Good luck with getting the issue resolved though.

**Comment by FranciscoAT** [comment]
Tl;dr I just need to restart it. Should be working now :)

**Comment by FranciscoAT** [comment]
Going to close this issue. Feel free to reopen the issue if things are still not working.

**Comment by shetaron** [comment]
Its back at it again
